Project.toml

```
@@ -3,7 +3,7 @@ uuid = "47edcb42-4c32-4615-8424-f2b9edc5f35b"
 authors = [
     "Vaibhav Dixit <vaibhavyashdixit@gmail.com>, Guillaume Dalle and contributors",
 ]
-version = "1.3.0"
+version = "1.3.1"
```

```suggestion
version = "1.4.0"
```

Should bump minor since it'll be a new feature